.storybook/stories/playground/index.js

```diff
@@ -17,44 +17,98 @@
 /**
  * External dependencies
  */
-import StoryEditor from '@web-stories-wp/story-editor';
+import StoryEditor, { InterfaceSkeleton } from '@web-stories-wp/story-editor';
+import styled from 'styled-components';
 
 export default {
   title: 'Playground/Stories Editor',
 };
 
-// @todo: Find better way to mock these.
+// @todo Instead of 100vh, may be the story editor should define its minimum required height to work properly,
+// and that height should be set with the <StoryEditor> component.
+const AppContainer = styled.div`
+  height: 100vh;
+`;
+
+const MediaUpload = ({ render }) => {
+  return render(() => {});
+};
+
+// @todo None of these should be required by default.
+const apiCallbacksNames = [
+  'getAuthors',
+  'getStoryById',
+  'getDemoStoryById',
+  'saveStoryById',
+  'autoSaveById',
+  'getMedia',
+  'getMediaById',
+  'getMutedMediaById',
+  'getOptimizedMediaById',
+  'uploadMedia',
+  'updateMedia',
+  'deleteMedia',
+  'getLinkMetadata',
+  'getCustomPageTemplates',
+  'addPageTemplate',
+  'deletePageTemplate',
+  'getCurrentUser',
+  'updateCurrentUser',
+  'getHotlinkInfo',
+  'getProxyUrl',
+  'getPublisherLogos',
+  'addPublisherLogo',
+  'getTaxonomies',
+  'getTaxonomyTerm',
+  'createTaxonomyTerm',
+];
+
+const apiCallbacks = apiCallbacksNames.reduce((callbacks, name) => {
+  let response;
+
+  switch (name) {
+    case 'getCurrentUser':
+      response = { id: 1 };
+      break;
+    case 'getStoryById':
+    case 'getDemoStoryById':
```

Comment:
I see, thanks. 

I am looking at the possibilities of using it, we also need something like that for loading the initial state of the story from `localStorage`.
> It might make more sense to have something like the `initialEdits`